### PR TITLE
Hyunku Kwon, hw submission

### PR DESCRIPTION
**Pull request by HyunkuKwon**
Actually, I almost had a mental breakdown in the process of familiarizing myself with GitHub. I forked the class repository. Then, I opened R Studio, clicked new project, version control, git, and entered the url of my repo. I opened the READ.md, which led to Rmarkdown page. I made adjustments in the Rmarkdown page. 

As I had no idea of the modus operandi of how to pull/push or commit, I had a really, really frustrating time. I tried to pull/push without committing, or to commit without saving the changes, both of which led to undecipherable messages including "non-fast-forwarding" whatever. I was really frustrated because I did not know what to do with this. Finally, I realized that I should save the changes first, and click the README button whenever I want to commit, and press the commit before I hit the push button. Unfortunately, I still cannot understand a lot. I have a long-long way to go for now. I found Stackoverflow helpful, but the posts in the website also came across so hard to understand for now. I googled some of the problems, but as I know nothing about Github, most of the advice were hard to understand, I just clicked whatever I can, the random and arbitrary process of which finally led me to the right (?) direction. Trial and error helps, but it takes so long...

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,13 @@
...
+
+My research interest lies in 
+1. Political Polarization
+2. Public Opinion
+3. Political Psychology
+
+Here is a image of me
+![](https://github.com/HyunkuKwon/hw01/blob/master/KakaoTalk_20171001_221540326.jpg)
+My facebook is as follows:
+[link](https://www.facebook.com/hyunku.kwon.9)
```
Comment (AymanBaig) [inline README.md:13]: I am not actually able to follow the link. However, I am not completely sure if the link as I am viewing it here is supposed to work.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,13 @@
...
+
+Hi! I am Hyunku Kwon, a _MAPSS_ student. I was born in **South Korea**. I graduated from Yonsei University, where I majored in Public Policy and Political Science, with a special focus on political liberalism and political economy of welfare state. I was an exchange student at University of Wisconsin-Madison. For the last two years, I served my military duty in South Korea. 
+
+My research interest lies in 
+1. Political Polarization
+2. Public Opinion
+3. Political Psychology
+
+Here is a image of me
+![](https://github.com/HyunkuKwon/hw01/blob/master/KakaoTalk_20171001_221540326.jpg)
```
Comment (AymanBaig) [inline README.md:11]: I do not see the picture here in this green box, but I see it in another section. I am not sure if the picture is supposed to appear here or not.

**Comment by DH1460** [comment]
Evaluation HW-xx for Jane Doe
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |   √        |              |            |
| **Coding strategy**         |   √       |              |            |
| **Presentation**            |   √      |              |            |
| **Achievement, creativity** |   √        |              |            |
| **Ease of access**          |   √        |              |            |

Well done! You covered 4 out of 5 elements, including _list_, _image_, _headline_, and _bold_ and _italic_.
I like how you list your research interests. The **list** element, _*1.*_, _*2*_, and _*3*_, in the project works very well. One thing I would add is that how you came cross to study political sciences! Overall, very well done!

**Review by AymanBaig — COMMENTED** [review]
Evaluation HW-01 for Hyunku Kwon
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |       X    |              |            |
| **Coding strategy**         |    X       |              |            |
| **Presentation**            |           |         X     |            |
| **Achievement, creativity** |     X      |              |            |
| **Ease of access**          |   X        |              |            |

Remarks:
* Elaborate on above, especially for "needs work."
* Some specific praise? Hyunku did a good job at executing the necessary coding requirements. I believe that we only had to do 4 things from the list in the homework description, but he did all 5. 
* Something I learned? I actually tried to insert a picture in my code as well, but I was really confused about it. I can use his code as a guide if I were to ever add a picture in the future. 
* Specific constructive criticism? I was not actually able to follow the link that he provided. However, as I commented, I am not sure if the link is supposed to work through the green text box. I am also not sure if the image is supposed to appear separately as it did in his code. 
* Something I know and that you, my peer, might like to know because it is relevant to something you struggled with. I was able to also use the R Markdown website for instructions on how to do the basic commands that were needed for this homework.

File: README.md (hunk 2 above)
Comment (HyunkuKwon) [reply README.md:11]: Thank you very much for your meticulous comments. and sorry for some of the meaningless commits that I made by mistake.

**Comment by sushmitavgopalan16** [comment]
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |        x   |              |            |
| **Coding strategy**         |      x     |              |            |
| **Presentation**            |      x     |              |            |
| **Achievement, creativity** |     x      |              |            |
| **Ease of access**          |      x     |              |            |

Remarks:

* Excellent work! Your page looks great - you've shown proficient use of various different elements.
* Your code is clean and clear. 
* Have you figured out your questions about Git and pushing/pulling/committing? Do come to office hours if you have any questions at all - remember the 15 minute rule!

**Pull request closed without merging**